tests/testthat/test-dist.R

```diff
@@ -108,3 +108,22 @@ test_that("test kullback kullback distance", {
         entropy::KL.empirical(dmat[,2] + 1, dmat[,1] + 1)
     )
 })
+
+test_that("test hamming distance", {
+    new_mat_test <- rsparsematrix(100, 80, 1, rand.x = function(x) sample.int(10, x, replace = TRUE))
```

Comment:
It is probably better to make the test matrix rectangular, but is there any reason to make `new_mat_test` 100 by 80 while `new_mat_test` 100 by 90?

Reply:
None at all. I'll set the other example to 100×90 as well.